homeassistant/components/nextcloud/strings.json

```diff
@@ -63,8 +63,14 @@
       "nextcloud_system_freespace": {
         "name": "Free space"
       },
-      "nextcloud_system_cpuload": {
-        "name": "CPU Load"
+      "nextcloud_system_cpuload_1": {
```

Comment:
Since these are new keys, we can already remove `nextcloud_`.

Reply:
i'm not sure if we should mix up the different key namings, instead of rename then in one batch within an own PR 🤔

Reply:
Agreed